### ISSUE TITLE
[Bug] Oct 26 Release Crashes on Launch

### ISSUE THREAD

**Issue by AronL** [arichorn/uYouPlusExtra#172]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### Device info

- iOS/iPadOS version: 17.1b1
- Device model: 15 Pro
- Sideload tool (AltStore, Sideloadly, TrollStore,...): Scarlet
- The specific version of uYouPlus (**latest or newest is NOT a version number!**): v18.41.5-3.0.1

### Describe the bug

When I install the app with Scarlet, it crashes.

### Steps to reproduce the issue

1. Install app with scarlet
2. App crashes on launch
3. Profit

### Are you using the newest version of uYouPlus? If not, why?

✅ Yes, I'm using the latest version of uYouPlus right now

### Does the issue happen with the official YouTube from AppStore?

❌ No, I can't reproduce this issue in the official YouTube from AppStore

### Additional context

![image](https://github.com/arichorn/uYouPlusExtra/assets/21107428/f3997b30-a7c9-4260-9060-595b99495959)

**Comment by haomuch** [comment]
> I’ll update uYouPlusExtra to v18.41.5. And by the way, that version you are currently using was not uYouPlusExtra. Sorry if it was confusing 😅

Could you also update the broken uYouPlus ipa? Because it enables VP9 codecs for all resolutions.

**Comment by SuperAmirhamza** [comment]
> update

Yes yes this is what I want

**Comment by SuperAmirhamza** [comment]
> 
> 
> 
> Could you also update the broken uYouPlus ipa? Because it enables VP9 codecs for all resolutions. 

Yes yes this is what I wanted and also could you please explain to me about what you mean about the broken uyouplus ipa there's this one video uploaded in 720p using VP9 codec and on normal youtube looks fine and on here looks like it's turned off
